src/components/dataSource/DataSource.tsx

```diff
@@ -0,0 +1,68 @@
+import { AvailableIcons, Icon, Input, Label, useTheme } from '@chili-publish/grafx-shared-components';
+import { useCallback, useEffect, useState } from 'react';
+import { ConnectorInstance, ConnectorType } from '@chili-publish/studio-sdk';
+import { PanelTitle } from '../shared/Panel.styles';
+import { getDataIdForSUI, getDataTestIdForSUI } from '../../utils/dataIds';
+
+function DataSource() {
+    const { panel } = useTheme();
+    const [dataConnector, setDataConnector] = useState<ConnectorInstance | null>();
+    const [firstRowInfo, setFirstRowInfo] = useState('');
```

Comment:
seems like it should be `currentRow` because you will have possibility to switch in follow up tickets